go/pkg/client/retries_test.go

```diff
@@ -287,6 +298,39 @@ func TestReadRetries(t *testing.T) {
 	}
 }
 
+func TestReadToFileRetries(t *testing.T) {
+	t.Parallel()
+	for _, sleep := range []bool{false, true} {
+		sleep := sleep
+		t.Run(fmt.Sprintf("sleep=%t", sleep), func(t *testing.T) {
+			t.Parallel()
+			f := setup(t)
+			defer f.shutDown()
+			if sleep {
+				f.fake.sleepDelay = time.Second
+				client.RPCTimeouts(map[string]time.Duration{"default": 500 * time.Millisecond}).Apply(f.client)
+			}
+
+			blob := []byte("blob")
+			path := filepath.Join(t.TempDir(), strings.Replace(t.Name(), "/", "_", -1))
+			n, err := f.client.ReadBlobToFile(f.ctx, digest.NewFromBlob(blob), path)
+			if err != nil {
+				t.Errorf("client.ReadBlobToFile(ctx, digest) gave error %s, want nil", err)
+			}
+			if n != 4 {
```

Comment:
`if n != len(blob)`, and in the error msg.

Reply:
Done.